Packs/CommonScripts/Scripts/ReadPDFFileV2/ReadPDFFileV2.py

```diff
@@ -149,7 +150,7 @@ def get_pdf_htmls_content(pdf_path, output_folder):
     for file_name in html_file_names:
         with open(file_name, 'rb') as f:
             for line in f:
-                html_content += line.decode('utf-8')
+                html_content += str(line)
```

Comment:
is this related?

Reply:
Sadly yes

Reply:
@amshamah419 but here you did not change the way you read the file. so how dose this change helps?

Reply:
Because the PDF body also contains broken encoding (not sure why). I sat on this for hours since I originally thought there was no fundamental difference between `str()` and .decode('utf-8)` 

`str()` when called without any arguments is the same as `object.__str__` which does not check for Python validity and will return a string object whether it is valid or not since the fallback is to return it as `repr()`

`.encode()` by default assumes the bytes to be utf-8 and handles errors strictly. So in this case, it fails because the mixed encoding isn't valid.

Reply:
Worth noting that if we use `.decode('utf-8', 'replace')` it will work also.

Reply:
Nice